app/controllers/planning_applications_controller.rb

```diff
@@ -289,6 +289,11 @@ def validate_documents
 
   private
 
+  def validation_date_fields_invalid?
+    validation_date_fields.any?(&:blank?) ||
+      validation_date_fields.any? { |field| !field.match(/\A[0-9]*\z/) }
```

Comment:
Prior to updating the `govuk_design_system_formbuilder` gem, an HTML5 validation handled this without submitting the form. Now the form overrides this, allowing us to use back-end validation and render a proper gov-uk styled error message ☺️

Reply:
Probably out of scope but we probably don't need this method https://github.com/unboxed/bops/blob/61908f34fc9013d9df22131ea02f2653b0f89311/app/controllers/planning_applications_controller.rb#L338 as date params should be handled by rails. I think I had to do similar for determination date and didn't need to do any hacky things. Feels like we are doing a bit much here to validate this date field but possibly out of scope for this pr anyway